hiob-ludolf-centre-for-ethiopian-studies-long-names.csl

```diff
@@ -0,0 +1,347 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Hiob Ludolf Centre for Ethiopian Studies (long names)</title>
+    <title-short>HLCES (Hamburg) long names</title-short>
+    <id>http://www.zotero.org/styles/hiob-ludolf-centre-for-ethiopian-studies-long-names</id>
+    <link href="http://www.zotero.org/styles/hiob-ludolf-centre-for-ethiopian-studies-long-names" rel="self"/>
+    <link href="http://www.zotero.org/styles/hiob-ludolf-centre-for-ethiopian-studies" rel="template"/>
+    <link href="http://www1.uni-hamburg.de/COMST/bulletin.html" rel="documentation"/>
+    <author>
+      <name>Eugenia Sokolinski</name>
+      <email>eugenia.sokolinski@uni-hamburg.de</email>
+    </author>
+    <contributor>
+      <name>Alessandro Bausi</name>
+      <email>alessandro.bausi@uni-hamburg.de</email>
+    </contributor>
+    <contributor>
+      <name>Pietro Liuzzo</name>
+      <email>pietro.liuzzo@uni-hamburg.de</email>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="humanities"/>
+    <updated>2016-09-10T19:36:18+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en-GB">
+    <terms>
+      <term name="ordinal">th</term>
+      <term name="ordinal-01">st</term>
+      <term name="ordinal-02">nd</term>
+      <term name="ordinal-03">rd</term>
+      <term name="ordinal-04">th</term>
+      <term name="ordinal-11">th</term>
+      <term name="ordinal-12">th</term>
+      <term name="ordinal-13">th</term>
```

Comment:
I reindented all three styles with http://formatter.citationstyles.org/, and changed the name of the new style slightly to be more consistent with the rest of our styles.

I think the "ordinal" terms in the locale section of all three styles can be removed. The "ordinal-04" term should be unnecessary for English, and all the other terms are already defined in the British English CSL locale file: https://github.com/citation-style-language/locales/blob/da68506f8f47089d28a8289db75f2f5b7f8e5000/locales-en-GB.xml#L71

Reply:
Thanks a lot! I do not know why they were put there in the first place to be honest. might have been inherited from the style used to begin the editing of the new one. I am removing them now.